README.md

```diff
@@ -181,13 +181,20 @@ Actions can be used for pre or post release parts.
     * Option `commit-message`: specify a custom commit message. %version% will be replaced by the current / next version strings.
 * `vcs-tag`: Tag the last commit
 * `vcs-publish`: Publish the changes (commits and tags)
-* `composer-update`: Update the version number in a composer file
-* `update-version-class`: Update the version constant in a class file.
-    * Option `class`: path to class to be updated, or fully qualified class name of the class containing the version constant
+* `update-file`: Update the version number in a file
+    * Option `file`: path to be updated
+    * Option `pattern`: optional, use to specify the string replacement pattern in your
+      version file. %version% will be replaced by the current / next version strings.
+      For example you could use `const VERSION = '%version%';`. If you do not specify
+      this option, every occurrence of the version string in the file will be replaced.
```

Comment:
For me this is an issue as you can have many situations where you don't have a current version. For example, when you start an empty project. Or if you are using the `vcs` persister, each time you release a new branch, there is no "current" version.